pkg/controller/watch/event_handler.go

```diff
@@ -2,10 +2,10 @@ package watch
 
 import (
 	"go.uber.org/zap"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
Same thing as before, haven’t touched this file but linter demanded I sort these imports.